pkg/bandwidth/bandwidth.go

```diff
@@ -74,6 +76,82 @@ func ProbeBandwidthManager() {
 	}
 }
 
+func setBaselineSysctls() error {
+	// Ensure interger type sysctls are no smaller than our baseline settings
+	baseIntSettings := map[string]int64{
+		"net.core.netdev_max_backlog":  1000,
+		"net.core.somaxconn":           4096,
+		"net.ipv4.tcp_max_syn_backlog": 4096,
+	}
+
+	for name, value := range baseIntSettings {
+		currentValue, err := sysctl.ReadInt(name)
+		if err != nil {
+			return fmt.Errorf("read sysctl %s failed", name)
+		}
+
+		scopedLog := log.WithFields(logrus.Fields{
+			logfields.SysParamName:  name,
+			logfields.SysParamValue: currentValue,
+			"baselineValue":         value,
+		})
+
+		if currentValue >= value {
+			scopedLog.Info("Skip setting sysctl as it already meets baseline")
```

Comment:
I would print current and baseline numbers here for debugging purposes

Reply:
(and the parameter name)

Reply:
@aspsk Thanks! Not sure if I understand it correctly, `scopedLog` already contains the sysctl parameter name, current value and baseline value.